### ISSUE TITLE
TotW 5 shows twice in sitemap

### ISSUE THREAD

**Issue by jonmeow** [abseil/abseil.github.io#370]
On https://abseil.io/tips/, I see:
TotW #3: String Concatenation and operator+ vs. StrCat()
TotW #5: Disappearing Act
TotW #5: Disappearing Act
TotW #10: Splitting Strings, not Hairs

i.e., TotW #5 shows twice. Seems like a mistake.

**Comment by manshreck** [comment]
Yep, that's because tip 5 is being generated twice, each with a different date.

Andy, what's the canonical date for this tip (original publication date?) I would assume it's the earlier one:

2020-06-01-totw-5.md
2020-09-11-totw-5.md

**Comment by axgillies** [comment]
Duplicate of https://github.com/abseil/abseil.github.io/issues/502.
